microsoft-365/enterprise/ms-cloud-germany-transition-add-experience.md

```diff
@@ -22,9 +22,9 @@ description: "Summary: Additional customer experience information when moving fr
 
 # Migration phases actions and impacts for the migration from Microsoft Cloud Deutschland (advanced) 
 
-Tenant migrations from Microsoft Cloud Deutschland to the Germany region of Microsoft's Office 365 services are executed as a set of phases and their configured actions for each workload. This figure shows the nine phases of migration to the new German datacenters.
+Tenant migrations from Microsoft Cloud Deutschland to the Germany region of Microsoft's Office 365 services are executed as a set of phases and their configured actions for each workload. This figure shows the ten phases of migration to the new German datacenters.
 
-![The nine phases of migration to the new Germany datacenters](../media/ms-cloud-germany-migration-opt-in/migration-organization.png)
+![The ten phases of migration to the new Germany datacenters](../media/ms-cloud-germany-migration-opt-in/migration-organization.png)
```

Comment:
The graphic needs to be updated to show 10 phases.

Reply:
This was included in the PR